src/test/dynamo/scan.test.ts

```diff
@@ -246,3 +255,26 @@ async function testScanWithLimit(jayZ?: JayZ) {
 
   expect(response2.items).toEqual({ musician: [], song: [song1] })
 }
+
+async function testScanWithPaginatedResultsReturnUndefinedCursor(jayZ?: JayZ) {
+  const db = await setup(jayZ)
+  const [musician, song1, song2] = aMusicianWithTwoSongs()
+  await Promise.all([db.put(musician), db.put(song1), db.put(song2)])
+
+  const { value: response1 } = await db.scan().iterator({ pageSize: 3 }).next()
+
+  expect(response1.items).toEqual({
+    musician: [musician],
+    song: [song1, song2]
+  })
+
+  const { value: response2 } = await db
+    .scan()
+    .iterator({
+      cursor: response1.cursor,
+      pageSize: 1
+    })
+    .next()
+
+  expect(response2.items).toEqual({ musician: [], song: [] })
```

Comment:
Let's also make sure the cursor is `undefined` here: `expect(response2.cursor).toBeUndefined()`.

Reply:
Ah yea, good catch